src/sentry/static/sentry/app/components/projects/missingProjectMembership.tsx

```diff
@@ -116,7 +116,11 @@ class MissingProjectMembership extends React.Component<Props, State> {
     const teams = this.state.project?.teams ?? [];
     teams.forEach(({slug}) => {
       const team = TeamStore.getBySlug(slug);
-      team?.isPending ? pending.push(team!.slug) : request.push(team!.slug);
+      if (!team) {
+        return;
+      } else {
```

Comment:
small nit but the else is not needed since we return in the other condition.

Reply:
oh right, fixed!